models/baseModels/InvoiceItem/InvoiceItem.ts

```diff
@@ -789,7 +828,7 @@ function getRate(
   const discountBeforeTax = !discountAfterTax;
 
   /**
-   * Rate calculated from  itemDiscountedTotal
+   * Rate calculated from itemDiscountedTotal
    */
```

Comment:
Comments aren't necessary unless it is very ambiguous for a second person to understand.